br/pkg/lightning/restore/get_pre_info.go

```diff
@@ -367,7 +367,9 @@ func (p *PreRestoreInfoGetterImpl) getTableStructuresByFileMeta(ctx context.Cont
 	dbName := dbSrcFileMeta.Name
 	currentTableInfosFromDB, err := p.targetInfoGetter.FetchRemoteTableModels(ctx, dbName)
 	if err != nil {
-		return nil, errors.Trace(err)
+		if !strings.Contains(err.Error(), "Unknown database") {
```

Comment:
maybe use the named error https://github.com/pingcap/tidb/blob/6a6fa8a882926eb402be866fb7e35ddf20f83c42/errno/errname.go#L73

Reply:
maybe add it into comment

Reply:
Done in https://github.com/pingcap/tidb/pull/36904/commits/6f32ad8911de39b7ca02cd5c9e1f66b6f06fc341